app/assets/stylesheets/bootstrap-headerfix.scss

```diff
@@ -41,11 +41,12 @@ header {
         font-weight: bold;
       }
     }
-    .notification_element {
+
+    .media.stream_element{
       font-size: 13px;
-      .timeago {
-        border: medium none;
-        cursor: text;
+      img.avatar {
+        width: 35px;
+        height: 35px;
```

Comment:
Please keep that in header.scss where it has been before and to make sure that it doesn't change the style of hovercards it might be a good idea to use something like `> .avatar` what we also had before. While touching this file I guess you could also remove `font-size: 13px;`. We already set the default font-size to 13px.